openwisp_controller/config/static/config/js/relevant_templates.js

```diff
@@ -88,85 +95,60 @@ django.jQuery(function ($) {
         }
       }
     },
+    getRelevantTemplateUrl = function (orgID, backend) {
+      // Returns the URL to fetch relevant templates
+      var baseUrl = window._relevantTemplateUrl.replace("org_id", orgID);
+      var url = new URL(baseUrl, window.location.origin);
+
+      // Get relevant templates of selected org and backend
+      if (backend) {
+        url.searchParams.set("backend", backend);
+      }
+      if (isDeviceGroup() && !$(".add-form").length) {
+        // Get the group id from the URL
+        // TODO: This is fragile, consider using a more robust way to get the group id.
+        var pathParts = window.location.pathname.split("/");
```

Comment:
We are overriding `/templates/admin/device_group/change_form.html`, so why don't we add the group ID there as an HTML element which is easy to retrieve here?